spacy/ml/models/multi_task.py

```diff
@@ -24,6 +26,80 @@ def build_cloze_multi_task_model(vocab, tok2vec):
     return model
 
 
-def build_masked_language_model(*args, **kwargs):
-    # TODO cf https://github.com/explosion/spaCy/blob/2c107f02a4d60bda2440db0aad1a88cbbf4fb52d/spacy/_ml.py#L828
-    raise NotImplementedError
+def build_masked_language_model(vocab, wrapped_model, mask_prob=0.15):
+    """Convert a model into a BERT-style masked language model"""
+
+    random_words = _RandomWords(vocab)
+
+    def mlm_forward(model, docs, is_train):
```

Comment:
Yeah this seems fine. The random state in the outer closure is a bit awkward and may inconvenience someone at some point, but let's leave it alone for now.